### PR TITLE
Add mobile-event JSON schema.

### PR DESCRIPTION
**Pull request by justindarc**

### CODE REVIEW

**Comment by justindarc** [comment]
@fbertsch

**Comment by fbertsch** [comment]
@justindarc please take a look at the readme, the new schemas go in to `templates/` where you then build the schemas. If possible, try making the `focus-event` schema a template, and using it for both `mobile-event` and `focus-event`.

**Comment by justindarc** [comment]
Sorry, my bad. I've updated the PR to put the new schema into `templates/` instead. How can I make one template be used for both ping formats?

**Comment by fbertsch** [comment]
@justindarc see application [0], which is a template and is used in the main ping [1]. You should be able to look at the resulting schema in `schemas/` and checking that it looks correct.

Please also add at least one validation ping.

[0] https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/master/templates/include/telemetry/application.1.schema.json
[1] https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/master/templates/telemetry/main/main.4.schema.json#L6

**Comment by fbertsch** [comment]
@justindarc please also rebase this change on top of `dev`, not `master`. Thanks!

**Comment by justindarc** [comment]
Done.

File: templates/telemetry/mobile-event/mobile-event.1.schema.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "mobile-event",
+  "properties": {
+    @TELEMETRY_MOBILEEVENT_1_JSON@
+  },
+  "required": [ "v", "clientId", "seq", "locale", "os", "osversion", "created", "settings", "events" ]
```
Comment (fbertsch) [inline templates/telemetry/mobile-event/mobile-event.1.schema.json]: Can we move the entirety of `properties` and `required` to the template?

**Comment by justindarc** [comment]
@fbertsch review comments have been addressed.

**Pull request merged**